### ISSUE TITLE
App notification crash on Motorola

### ISSUE THREAD

**Issue by knighthat** [fast4x/RiMusic#4833]
### Version of RiMusic

compiled straight from source

### Android version

10

### I've done these:

- [X] Uninstall old build before installing a new one
- [X] Checked opening issues about my similar problem

### Tell us how you trigger this bug

Play any song

### Submit a video

https://github.com/user-attachments/assets/a94da004-496e-445f-9e2d-4a7d9803bc98

### What was your expectation?

play normally

### What more can you tell us? (Optional)

Possibly related to #4831 #4829.

I thought it was my PR #4822 but the recorded version was reverted back to before my commits.

### Logs (Optional)

> Straight from IDE

FATAL EXCEPTION: 

Process: it.fast4x.rimusic.debug, PID: 5520

```
android.app.RemoteServiceException: Bad notification for startForeground
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2093)
    at android.os.Handler.dispatchMessage(Handler.java:107)
    at android.os.Looper.loop(Looper.java:241)
    at android.app.ActivityThread.main(ActivityThread.java:7582)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:941)
```

**Comment by fast4x** [comment]
Fixed, available from version 0.6.63.1

**Comment by knighthat** [comment]
thanks, I didn't have much time to look into it
